qml/ScheduleClearDialog.qml

```diff
@@ -53,6 +55,78 @@ Dialog {
         columns: 2
         rowSpacing: 20
 
+
+        Label {
+            text: "Dump"
+            InfoToolTip {
+                text: "Dump database to a file."
+            }
+        }
+        CheckBox {
+            id: dump
+            checked: false
+            indicator.width: 20
+            indicator.height: 20
+            onCheckedChanged: {
+                if (checked) {
+                    dumpOn = true
+
+                } else {
+                    dumpOn = false
+                }
+            }
+        }
+
+        Label {
+            text: "Select path:"
+            enabled: dumpOn
+            InfoToolTip {
+                text: "Path to save the dump \n"+
+                        "file."
+            }
+        }
+        Button {
+            text: "Choose"
```

Comment:
I would add one of these options to make user experience easier
1. write next to the button the current path chosen (preferably)
2. when clicking the button again, move to the last path chosen